hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/ozoneimpl/TestBackgroundContainerDataScanner.java

```diff
@@ -104,6 +104,12 @@ public void testScannerMetrics() {
     assertEquals(1, metrics.getNumUnHealthyContainers());
   }
 
+  @Test
+  public void testTotalRunTimes() {
+    scanner.runIteration();
+    assertTrue(scanner.getMetrics().getTotalRunTimes() > 0);
```

Comment:
We can make this a little more robust by modifying one of the container mocks to have a small sleep during its scan. The mocked containers are directly accessible from this test, so you could make it so that when one of the container's `scanData` methods is called it will sleep for 500ms. Then this test would check that the total run time is > 500ms. We can then run the scan again and check that it goes up to > 1 second. These are just example numbers, we could probably use lower values.

Reply:
Same for the metadata test.

Reply:
Also, can we add this to the existing metrics test instead? Currently this is the only metric tested outside of that method.